doneill/er-cli-go
# user command impl

- Refactor/Modularize **earthranger_service** to **ersvc**
- Modularize api functions into own source files
- Create a `user` command
- Display full response

**Manual Test**
- Authenticate user
- Run `./er user`
- Confirm results match authenticated user
